POLICY.rst

```diff
@@ -0,0 +1,104 @@
+=================================
+Salt Project maintenance policies
+=================================
+
+This document explains the current project maintenance policies. The goal of
+these policies are to reduce the maintenance burden on core maintainers of the
+Salt Project and to encourage more active engagement from the Salt community.
+
+* `Issue management`_
+* `Pull request management`_
+* `Salt Enhancement Proposals (SEP) process`_
+
+
+Issue management
+================
+Issues for the Salt Project are critical to Salt community communication and to
+find and resolve issues in the Salt Project. As such, the issue tracker needs to
+be kept clean and current to the currently supported releases of Salt. They also
+need to be free of feature requests, arguments, and trolling.
+
+We have decided to update our issue policy to be similar to RedHat community
+project policies.
+
+Community members who repeatedly violate these policies are subject to bans.
+
+#. All issues that were not opened against a currently supported release of Salt
+   will be closed.
+
+   - When an old release of Salt is marked out of support, all issues opened
+     against the the now defunct release will be closed.

```

Comment:
```suggestion
     against the now defunct release will be closed.

```

Reply:
Thanks for the catch! I made this change on my own branch and pushed it up before I realized this was a commit suggestion. I tend to prefer to work in my own branch anyhow. It should be resolved now!